lib/ClimaCoreTempestRemap/src/ClimaCoreTempestRemap.jl

```diff
@@ -150,4 +150,30 @@ function write_exodus(
 end
 
 
+function rll_mesh(filename::String; nlat = 90, nlong = round(Int, nlat * 1.6))
```

Comment:
return `filename` / `outfile` on success?

Reply:
this was supposed to be a separate PR, I'll remove it.